src/views/Projects/index.jsx

```diff
@@ -17,14 +20,37 @@ import ProjectCard from '../../components/ProjectCard';
     paddingBottom: 2 * theme.spacing.unit,
   },
 }))
-export default class Projects extends Component {
+export default class Index extends Component {
```

Comment:
What is the reason for this change?

Reply:
initially we have 2 file in Projects : index.jsx and ProjectsView.jsx , index only has the routing (/projects and /projects/:projectKey),  then after rebase, i realize I can move the routing to the App.jsx since now it is (/ and /:projectKey)